pkgs/by-name/op/openvsp/package.nix

```diff
@@ -0,0 +1,114 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  git,
+  pkg-config,
+  python3,
+  eigen,
+  fltk,
+  glm,
+  glew,
+  cminpack,
+  libxml2,
+  graphviz,
+  lib,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "openvsp";
+  version = "3.40.1";
+
+  src = fetchFromGitHub {
+    owner = "OpenVSP";
+    repo = "OpenVSP";
+    rev = "OpenVSP_${version}";
+    hash = "sha256-PfXqnd75D06vER7X5w+VOm12mvLbcSe4sxmhtwBmPps=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    python3
+    git
+  ];
+
+  # swig & doxygen are not included as the build would fail since it tries to call
+  # "swig -doxygen" which fails Make as this is not a valid command.
+  # Seems like an upstream problem.
+  buildInputs = [
+    cminpack
+    eigen
+    fltk
+    glew
+    glm
+    graphviz
+    libxml2
+  ];
+
+  configurePhase = ''
+    mkdir -p build buildlibs
+
+    pushd buildlibs
+
+    cmake \
```

Comment:
Can we use our default cmake configure phase with cmakeFlags?

Reply:
The build breaks when the cmakeFlags' from nix are concatenated.